src/pydantic2linkml/cli/__init__.py

```diff
@@ -0,0 +1,35 @@
+from pathlib import Path
+
+import typer
+from linkml_runtime.dumpers import yaml_dumper
+
+from pydantic2linkml.gen_linkml import LinkmlGenerator
+from pydantic2linkml.tools import fetch_defs, get_all_modules
+
+app = typer.Typer()
+
+
+@app.command()
+def main(
+    module_names: list[str],
+    output_file: Path = None,
+):
+    # TODO: RF prints to log messages to stderr
+    modules = get_all_modules(module_names)
+    print(
```

Comment:
replace all diagnostic etc prints of this kind to be logger messages and logger should log to stderr.

Reply:
Done.